New year, new look!

wjhsf opened the pull request:
<!--- Provide a short summary of your changes in the Title field above -->

# Description

<!--- A longer summary of your changes, including: a description of the issue that you’re addressing, a list of required dependencies (if applicable), and any other relevant context. -->

# Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] **Bug fix** (non-breaking change that fixes an issue)
- [ ] **New feature** (non-breaking change that adds functionality)
- [ ] **Documentation update**
- [ ] **Breaking change** (could cause existing functionality to not work as expected)
- [ ] **Other changes** (non-breaking changes that does not fit any of the above)

> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- (step1)

# Checklists

<!--- Enter an `x` in all the boxes that apply. -->
<!--- If you’re unsure about any of these, don’t hesitate to ask. We’re here to help! -->

## General

- [ ] Changes are covered by test cases
- [ ] CHANGELOG.md updated with a short description of changes (_not_ required for documentation updates)

## Accessibility Compliance

You must check off all items in **one** of the follow two lists:

- [ ] There are no changes to UI

_or..._

- [ ] Changes were tested with a Screen Reader (iOS VoiceOver or Android Talkback) and had no issues
- [ ] Changes comply with [WCAG 2.0 guidelines levels A and AA](https://www.wuhcag.com/wcag-checklist/)
- [ ] Changes to common UI patterns and interactions comply with [WAI-ARIA best practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## Localization

- [ ] Changes include a UI text update in the Retail React App (which requires translation)

wjhsf committed:
New year, new look!

johnboxall commented on .eslintrc.js:
```diff
@@ -8,7 +8,7 @@ module.exports = {
                 '',
                 {
                     pattern: '^ \\* Copyright \\(c\\) \\d{4}, (salesforce.com, inc|Salesforce, Inc)\\.$',
-                    template: ' * Copyright (c) 2022, Salesforce, Inc.'
```
@wjhsf since this is a JS file, can we use JS here to always put in the correct year?

wjhsf replied:
🧠

wjhsf committed:
Merge branch 'develop' into wjh/2023

wjhsf committed:
Use JS to compute current year.

wjhsf committed:
Small bump to max deps allowed.

Needed to so that non-code changes will pass CI.
Hitting the limit can be addressed later.

wjhsf commented on .circleci/config.yml:
```diff
@@ -174,7 +174,7 @@ commands:
           environment:
           command: |
-            MAX_PACKAGES="2250"
+            MAX_PACKAGES="2260"
```
Hitting this limit in CI due to unrelated changes in nested deps. Proper remediation to be discussed later.
